js/index.js

```diff
@@ -11,7 +11,21 @@ webix.ready(function(){
   ];
 
   webix.ui({
+    view:"popup",
+    id:"profile_popup",
+    body:{
+      view:"list",
+      template:"#title#",
+      autoheight:true,
+      autowidth:true,
```

Comment:
autowidth does not work for List component, only for Menu
for List you need to define some fixed width (by default popups are 300px)

Reply:
Was edited

Reply:
but autoheight is ok here. without it the list is too high

Reply:
Was modified